### PR TITLE
20.7.0 rns

### PR DESCRIPTION
**Pull request by jochman**
## Status
Ready/In Progress/In Hold (Reason for hold)

### CODE REVIEW

File: content-repo/extra-docs/releases/20.7.0.md [hunk 1]
```diff
@@ -0,0 +1,516 @@
+# Cortex XSOAR Content Release Notes for version 20.7.0 (61089)
+##### Published on 07 July 2020
+
+---
+
+Welcome to the 20.6.1 Content release for Cortex XSOAR. Starting from the 20.6.0 release, we restructured our release notes to be based upon **Content Packs**. One of our team's top priorities is making our Content more accessible and understandable for both users and contributors. In this effort, we recently moved our [Content repo](https://github.com/demisto/content) to work in Packs format were there is a clear separation and grouping of Content artifacts. Each Content Pack provides a clear grouping of related Content artifacts used to either implement a use case, implement an integration or provide a clear set of functionality. Our new release notes are structured around Content Packs and you will see related Content artifacts grouped together according to Packs. We hope you will find this new format useful and clear.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.7.0.md:6]:
```suggestion
Welcome to the 20.7.0 Content release for Cortex XSOAR. Starting from the 20.6.0 release, we restructured our release notes to be based upon **Content Packs**. One of our team's top priorities is making our Content more accessible and understandable for both users and contributors. In this effort, we recently moved our [Content repo](https://github.com/demisto/content) to work in Packs format were there is a clear separation and grouping of Content artifacts. Each Content Pack provides a clear grouping of related Content artifacts used to either implement a use case, implement an integration or provide a clear set of functionality. Our new release notes are structured around Content Packs and you will see related Content artifacts grouped together according to Packs. We hope you will find this new format useful and clear.
```

File: content-repo/extra-docs/releases/20.7.0.md [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+### End Of Life Notice 
+The following Integrations were deprecated on Nov 2019:  
+- **Azure Compute**
+- **Azure Security Center**
+
+These integrations will reach the end of life on July 31, 2020, due to changes to the backend authentication services needed for these Integrations. Use the **Azure Compute v2** and **Azure Security Center v2** integrations instead.
+
+---
+
+### New: CrowdStrike FalconX Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.7.0.md:19]:
```suggestion
### New: CrowdStrike Falcon X Pack v1.0.0
```

File: content-repo/extra-docs/releases/20.7.0.md [hunk 3]
```diff
@@ -0,0 +1,516 @@
...
+
+### Bambenek Consulting Feed Pack v1.0.1
+#### Integrations
+##### Bambenek Consulting Feed  
+Added the *Tags* parameter.
+
+---
+
+### Base Pack v1.1.0
+#### CommonServerPowerShell
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:122]: Remove this line. Seems to have been added by mistake

File: content-repo/extra-docs/releases/20.7.0.md [hunk 4]
```diff
@@ -0,0 +1,516 @@
...
+##### SetGridField  
+Fixed an issue where the script failed on `mixed-types` error.
+
+##### JSONtoCSV  
+Improved error message when invalid JSON entry is given.
+
+---
+
+### Common Types Pack v1.4.0
+#### IncidentFields
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:184]: remove this line

File: content-repo/extra-docs/releases/20.7.0.md [hunk 5]
```diff
@@ -0,0 +1,516 @@
...
+#### IndicatorFields
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:185]: remove this line

File: content-repo/extra-docs/releases/20.7.0.md [hunk 6]
```diff
@@ -0,0 +1,516 @@
...
+#### Layouts
+##### domainRepUnified
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:187]: Can we get friendly names here? Such as `Domain Indicator`. @yaakovi can the sdk do this as part of the update release notes task?

File: content-repo/extra-docs/releases/20.7.0.md [hunk 7]
```diff
@@ -0,0 +1,516 @@
...
+Added the Feed Related Indicators section to the layout.
+
+##### urlRep  
+Added the Feed Related Indicators section to the layout.
+
+---
+
+### EWS Pack v1.1.3
+#### Integrations
+##### EWSO365
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:203]:
```suggestion
##### New: EWSO365  
```
@yaakovi fyi

File: content-repo/extra-docs/releases/20.7.0.md [hunk 8]
```diff
@@ -0,0 +1,516 @@
...
+New playbook Palo Alto Networks - Endpoint Malware Investigation v2
+
+---
+
+### PCAP Analysis Pack v2.1.0
+#### Playbooks
+##### New: PCAP Search  
+This playbook is used to parse and search within PCAP files.
+
+##### PCAP Parsing And Indicator Enrichment.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:279]:
```suggestion
##### New: PCAP Parsing And Indicator Enrichment.  
```
@yaakovi fyi

File: content-repo/extra-docs/releases/20.7.0.md (hunk 7 above)
Comment (Itay4) [reply content-repo/extra-docs/releases/20.7.0.md:203]: is this new? didnt we release it last time?

File: content-repo/extra-docs/releases/20.7.0.md [hunk 9]
```diff
@@ -0,0 +1,516 @@
...
+#### Integrations
+##### Remedy On-Demand  
+Fixed an issue where the ***remedy-incident-update*** and the ***remedy-get-incident*** commands required the request ID instead of the entry ID.
+
+---
+
+### RiskSense Pack v1.0.1
+#### Integrations
+##### RiskSense  
+added new command risksense-apply-tag that is a part of playbook for applying tag.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:377]:
```suggestion
Added new command risksense-apply-tag that is a part of playbook for applying tag.
```

File: content-repo/extra-docs/releases/20.7.0.md [hunk 10]
```diff
@@ -0,0 +1,516 @@
...
+Added the *Tags* parameter.
+
+---
+
+### ThreatConnect Pack v2.0.1
+#### Integrations
+##### ThreatConnect  
+Deprecated. Use the ThreatConnect v2 integration instead
+
+##### ThreatConnect v2
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:422]:
```suggestion
##### New: ThreatConnect v2  
```
@yaakovi fyi. Also can we add a line or two specifying what is new in v2 compared to v1.

File: content-repo/extra-docs/releases/20.7.0.md (hunk 6 above)
Comment (jochman) [reply content-repo/extra-docs/releases/20.7.0.md:187]: There's no friendly name in layouts, that's the biggest problem.

File: content-repo/extra-docs/releases/20.7.0.md (hunk 7 above)
Comment (jochman) [reply content-repo/extra-docs/releases/20.7.0.md:203]: yes, it is. removing

File: content-repo/extra-docs/releases/20.7.0.md [hunk 11]
```diff
@@ -0,0 +1,516 @@
...
+
+##### California - Breach Notification  
+Added setincidents to the playbook for the new layout.
+
+##### Residents Notification - Breach Notification  
+Added setincidents to the playbook for the new layout.
+
+---
+
+### Zoom Feed Pack v1.1.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:482]: I think this whole pack is new and needs to be moved to the top with the New stuff.

File: content-repo/extra-docs/releases/20.7.0.md [hunk 12]
```diff
@@ -0,0 +1,516 @@
...
+#### Integrations
+##### Zoom Feed  
+You can now use the new zoom site configuration using the feed.
+
+---
+
+### Zscaler Pack v1.0.2
+#### Integrations
+##### Zscaler  
+Added a *multiple* argument to the ***url*** command, which when set to "false" enables users to submit singular URLs that contain commas.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:492]: Use bullets

File: content-repo/extra-docs/releases/20.7.0.md [hunk 13]
```diff
@@ -0,0 +1,516 @@
...
+Improved list handling for the ***zscaler-category-add-url*** and ***zscaler-category-add-ip*** commands.
+
+---
+
+### iDefense Pack v1.0.1
+#### Integrations
+#### iDefense  
+Fixed an issue where the *Set API token* parameter was visible in the integration configuration window.
+---
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.7.0.md:501]: remove this line. It is causing the above line to be treated as title:

image

Comment (jochman) [reply content-repo/extra-docs/releases/20.7.0.md:501]: added white space instead

File: content-repo/extra-docs/releases/20.7.0.md (hunk 6 above)
Comment (glicht) [reply content-repo/extra-docs/releases/20.7.0.md:189]: Leave it for now.

**Pull request merged**